### ISSUE TITLE
`alt + b e` don't work as expected in Vue.js

### ISSUE THREAD

**Issue by QingCT** [Box-Of-Hats/Bem-VSCode-Extension#7]
### BEM Helper Version

0.7.0

### Steps to reproduce

First I create a new file: index.html

```
...
  <div class="nav-bar" :class="{'is-hidden':isHidden}">
    //
  </div>
...
```

Then I press `alt + b e` to insert element.

### What is Expected?

```
...
  <div class="nav-bar" :class="{'is-hidden':isHidden}">
    <div class="nav-bar__"></div>
  </div>
...

```

### What is actually happening?

```
...
  <div class="nav-bar" :class="{'is-hidden':isHidden}">
    <div class="{__"></div>
  </div>
...
```

### details

I rolled back to version `0.6.6` or changed the code like this and it works fine.

```
...
  <div :class="{'is-hidden':hidden}" class="nav-bar">
    //
  </div>
...
```

Thank you for making such a great plugin!

**Comment by Box-Of-Hats** [comment]
Thanks for giving such a detailed report. Ill look at this over the weekend

**Comment by Box-Of-Hats** [comment]
This issue is now fixed and released in version 0.7.1.

[Fixed in this commit](https://github.com/Box-Of-Hats/Bem-VSCode-Extension/commit/ff7713092bf1fb941cf724cea5c40c445359f9eb)
